osm2lanes/src/test.rs

```diff
@@ -34,6 +32,9 @@ pub struct TestCase {
     pub comment: Option<String>,
     pub description: Option<String>,
 
+    // Examplary
```

Comment:
```suggestion
    // List as a named example in the web app
```
unless we don't want to tie it so explicitly to that one use. (But it is otherwise a bit unclear why `description` is different)